pages/index.js

```diff
@@ -1,69 +1,14 @@
 import Head from 'next/head'
-import Image from 'next/image'
+import Footer from '../components/Footer'
+import Navbar from '../components/Navbar'
```

Comment:
I believe these are no longer used, and could be removed.

Your editor will often help you find unused variables, especially if you set up "linting". Getting rid of them helps readers understand what is important (everything left in the file) and what isn't (nothing, because if it's not important it should be taken away).